.ado/azure-pipelines.publish.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     pool:
       vmImage: 'macos-10.15'
       demands: ['xcode', 'sh', 'npm']
-    timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+    timeoutInMinutes: 90 # how long to run the job before automatically cancelling
```

Comment:
Having some data on why we need to increase this would be valuable. How long are the arm64e runs taking? How do they compare to the other architectures?

Reply:
It's a little iffy with differences per run (runs can be 2x longer just based on which agent it arbritraily got). I'll try and compile some simple numbers to compare.

Reply:
A good publish run is about 40 minutes. A bad publish run is about 60 minutes.
Adding the new architectures made the last run 46 minutes (6 minutes longer).
I estimate that a bad run would then be 75 minutes at most (60 + 15 minutes).
With that, I feel 90 is a good upper bound.